var/spack/repos/builtin/packages/py-pyarrow/package.py

```diff
@@ -15,6 +15,7 @@ class PyPyarrow(PythonPackage):
     homepage = "http://arrow.apache.org"
     url      = "https://pypi.org/packages/source/p/pyarrow/pyarrow-0.9.0.tar.gz"
 
+    version('0.12.1', sha256='10db6e486c918c3af999d0114a22d92770687e3a6607ea3f14e6748854824c2a')
```

Comment:
I see these are python bindings for the other package. Does it make sense to bind the versions like:
```python
depends_on('arrow@0.12.1', when='@0.12.1')
depends_on('arrow@0.11.0', when='@0.11.0')
depends_on('arrow@0.9.0', when='@0.9.0')
```
?

Reply:
It would make sense, but would add twice the lines at the moment. I'll see if that can be somewhat automated.